automated_testing/test/pageobjects/index-page.ts

```diff
@@ -91,19 +87,19 @@ class IndexPage extends Page {
         return $(selectors.pageContestCardTimerSelector);
     }
 
-    public get competeCardButtonActivecontests() {
+    public get competeButtonInActiveContestCard() {
         return $(selectors.pageAllActiveCardsSelector).$$(selectors.pageCompeteButtonIdSelector);
```

Comment:
Won't this return all `compete buttons`? If so, the name or the indent should be changed, i.e.
- If you want a single button, change the selection
- If you want all buttons, change the name

Reply:
I will change the name